scipy/special/basic.py

```diff
@@ -1616,6 +1616,10 @@ def lqn(n, z):
     Compute sequence of Legendre functions of the second kind, Qn(z) and
     derivatives for all degrees from 0 to n (inclusive).
 
+    Warning: The computation of lqn(n,z) could fail for large n. Specifically,
```

Comment:
There is a warning Sphinx directive that would be useful here--see e.g.

https://github.com/scipy/scipy/blob/master/scipy/special/__init__.py#L435